app/models/book.rb

```diff
@@ -5,6 +5,8 @@ class Book < ActiveRecord::Base
   # NOTE: The ISBN field in the database refers to a standard number; sometimes this is an ISBN.
   include CatalogItemMethods
   include Oauth
+  include Exceptions
```

Comment:
need to include  Exceptions module in book.rb. 
Otherwise below code will give the error.
raise Exceptions::InvalidResponse, "Invalid status code of: #{response.code}"